src/pages/blog/starlingx-subcloud-local-install.md

```diff
@@ -0,0 +1,65 @@
+A typical edge cloud deployment may consist of multiple controllers and potentially compute and storage nodes, connected over a network that is designed per the user’s specifications. This implicitly means that there are potential multiple points of delay and latency for network traffic.
+When there are packet losses during this data transfer, as may be the case depending on the network conditions at deployment time, then the system attempts to correct the losses through retransmissions, thereby increasing the deployment duration. If the data transfer is interrupted, for some reason, then the deployment may fail, and the server might require manual intervention.
+
+Now imagine a system where there are over a thousand edge clouds and multiple system controllers. It is obvious, therefore, that the cost of deployment failures is high, especially since the entire process must be restarted after a failure, and network issues are very difficult to debug. 
+The Subcloud Local Install feature aims to mitigate these issues and improve the deployment success rate.
+
+## What Happens when edge clouds are deployed
+
+To deploy an edge cloud, the installation artifacts must be made available to the physical machine. The most optimal way to do this is to provide them over the network. However, the size of these artifacts is over 3 GBs. This is not a problem for a very small system.
+
+However, , StarlingX supports larger numbers of edge clouds, as illustrated in Figure 2.
+
+[StarlingX Edge Cloud Improvements](/img/StarlingX_EdgeCloud_Improvements)
+
+As mentioned earlier, transferring huge amounts of data simultaneously to numerous remote machines in one go will stress the network and potentially quality compromise network quality of service. This forms the motivation for an enhancement in the way we approach the problem of deployment.
+
+## What if?
+What if the artifacts could be made available on the edge cloud already, when the System Controller begins deployment? In other words, can we avoid downloading artifacts at the time of deployment?
+The answer is yes.
+Deployment over the edge clouds can be enhanced, if the artifacts for installation were already present on the physical machines that make up the edge clouds, prior to the deployment operations, in persistent storage. This, in essence, is local install.
+Local Install consists of three scenarios. The first is where a physical machine comes “prestaged” with installation artifacts right from the factory; the second is when the user wants to upgrade an edge cloud, and the third, when the user wants to reinstall.
+
+### Factory Installation:
+The artifacts for installation are copied to a persistent storage location on the machine at the factory. When adding an edge cloud, the system utilizes the prestaged artifacts instead of downloading them.
```

Comment:
```suggestion
The installation artifacts are copied to a persistent storage location on the machine at the factory. When adding an edge cloud, the system utilizes the pre-staged images instead of downloading them from a remote location.
```